### ISSUE TITLE
Empty response when scheduling downtime for non existent host

### ISSUE THREAD

**Issue by widhalmt** [Icinga/icinga2#7631]
Hi,

Affected versions: icinga2-2.11.2

When scheduling an downtime via API for a non existing host Icinga doesn't give an understandable error response, but just replies with an empty value and response code 500. It would be better to have something like "host does not exist".

```
[root@monitor02 ~]# curl -k -s -w "%{http_code}" -u root:*** -H 'Accept: application/json' -X POST 'https://127.0.0.1:5665/v1/actions/schedule-downtime' -d '{"type":"Host","filter":"host.name==\"non.existing.host\"","start_time":"1573843056","end_time":"1573853056","author":"icingaadmin","comment":"test","fixed":true}'
{"results":[]}500
[root@monitor02 ~]# curl -k -s -w "%{http_code}" -u root:*** -H 'Accept: application/json' -X POST 'https://127.0.0.1:5665/v1/actions/schedule-downtime' -d '{"type":"Host","filter":"host.name==\"existing.host\"","start_time":"1573843056","end_time":"1573853056","author":"icingaadmin","comment":"test","fixed":true}'
{"results":[{"code":200.0,"legacy_id":6.0,"name":"existing.host!f5d503b5-b7e2-4da0-83cb-2f2a75132e7b","status":"Successfully scheduled downtime 'existing.host!f5d503b5-b7e2-4da0-83cb-2f2a75132e7b' for object 'existing.host."}]}200 
[root@monitor02 ~]# rpm -q icinga2
icinga2-2.11.2-1.el7.icinga.x86_64
```

**Comment by widhalmt** [comment]
ref/NC/631975

**Comment by widhalmt** [comment]
I can reproduce this in a testing environment and at a customers setup

**Comment by dnsmichi** [comment]
Should be fixed with #7528.

**Comment by widhalmt** [comment]
Thanks a lot.

**Comment by Al2Klimov** [comment]
@widhalmt Please test v2.12-rc1.

**Comment by Al2Klimov** [comment]
_PING_ @widhalmt

**Comment by Al2Klimov** [comment]
> Should be fixed with #7528.
